hack/ci/e2e.sh

```diff
@@ -153,29 +148,33 @@ run_tests() {
     -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.taints}{"\n"}{end}' \
     | grep -cv "node-role.kubernetes.io/master" )"
 
-  # setting this env prevents ginkg e2e from trying to run provider setup
+  # setting this env prevents ginkgo e2e from trying to run provider setup
   export KUBERNETES_CONFORMANCE_TEST="y"
-  # run the tests
```

Comment:
pointless comment, if you don't know what this script is, it's probably best to go take a look at it, as it has plenty of surprising properties...